app/assets/javascripts/miq_application.js

```diff
@@ -1308,7 +1308,7 @@ function miqSelectPickerEvent(element, url, options) {
 
   $('#' + element).on('change', _.debounce(function() {
     var selected = $(this).val();
-    var finalUrl = url + (firstarg ? '?' : '&') + element + '=' + escape(selected);
+    var finalUrl = url + (firstarg ? '?' : '&') + element + '=' + encodeURI(selected);
```

Comment:
`encodeURIComponent` should be called here see http://stackoverflow.com/questions/75980/when-are-you-supposed-to-use-escape-instead-of-encodeuri-encodeuricomponent/39444037#39444037 with a nice table demonstrating the difference